WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerViewModel.cs

```diff
@@ -114,7 +119,7 @@ private WalletViewModelBase OpenWallet(Wallet wallet, bool receiveDominant, bool
 
 			InsertWallet(walletViewModel);
 
-			walletViewModel.OpenWallet(receiveDominant);
+			walletViewModel.OpenWallet();
```

Comment:
Move here the logic.
https://github.com/zkSNACKs/WalletWasabi/pull/3332/files?file-filters%5B%5D=.cs#diff-184f9cf619f1d99781deb2d191bf8a07R122

Reply:
done